angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts

```diff
@@ -441,6 +456,17 @@ export class MinesCollectionsAddEditComponent implements OnInit, OnDestroy {
     }
   }
 
+  updateRecordList(recordToAdd) {
+    // tslint:disable
+    let formArray = this.myForm.get('collectionRecords') as FormArray;
```

Comment:
Should be able to get rid of the tslint:disable and make formArray a const. 
`const formArray = this.myForm.get('collectionRecords') as FormArray;`

Reply:
e942846